moveit_ros/moveit_servo/src/collision_check.cpp

```diff
@@ -203,6 +207,24 @@ void CollisionCheck::run(const ros::TimerEvent& timer_event)
   }
 }
 
+void CollisionCheck::printCollisionPairs(collision_detection::CollisionResult::ContactMap& contact_map)
+{
+  if (!contact_map.empty())
+  {
+    // Throttled error message about the first contact in the list
+    ROS_ERROR_STREAM_THROTTLE_NAMED(ROS_LOG_THROTTLE_PERIOD, LOGNAME, "Objects in collision (among others, possibly): "
```

Comment:
Is this really an error or should it be a warning?

Reply:
I wouldn't want to see people filtering out all debug info if this gets spammy. I think a rosparam for verbose collision checking would be a nice addition.

Reply:
If you modify the `collision_request_.contacts = verbose_collision_checking_` (where `verbose_collision_checking_` is set from a rosparam) than maybe `if (!contact_map.empty())` will always be false and you won't have to add additional logic in this method. Something to investigate

Reply:
will change it to a warning.

Not sure about making in configurable. It's already throttled so it doesn't get printed very often, and we already have a bunch of parameters to wade through (~40 of them)